src/node_dependencies/nested.js

```diff
@@ -60,4 +65,17 @@ const isOptionalModule = function (
   )
 }
 
+// 'critters' is used only in Next.js >= 10.0.4 when enabling an experimental option and has to be installed manually
+// we ignore it if it's missing
+const isExternalModule = function (moduleName, { dependencies = {}, devDependencies = {} }) {
+  if (moduleName !== 'critters') {
+    return false
+  }
+  const nextVersion = dependencies.next || devDependencies.next
+  if (!validRange(nextVersion)) {
```

Comment:
`satisfies()` returns `false` if `nextVersion` is not a valid version, so this can probably be removed?